scripts/e2e/detox-utils.cjs

```diff
@@ -1,6 +1,8 @@
 const ChildProcess = require('node:child_process');
+const { iosDevice } = require('./ios-devices');
 
-const CI_AVD_NAME = 'e2e_emulator';
+// Should be kept in sync with the constant defined in e2e workflow file
+const DEFAULT_CI_AVD_NAME = 'e2e_emulator';
```

Comment:
nit: would it be possible to get this value automatically? This way, we wouldn't need to remember to synchronize it? I'm seeing how it was working prior to this PR, so I'd consider it only as an improvement, which might be done in a separate task, but it's not a high priority and won't block this PR from merging.